icu4c/source/data/mappings/ibm-1388_P100-2024.ucm

```diff
@@ -20,6 +13,14 @@
 <icu:charsetFamily>           "EBCDIC"
 <icu:alias>                   "ibm-1388_VPUA"
 
+# includes mappings for surrogate pairs
+<icu:state>                   0-ff, e:1.s, f:0.s
+<icu:state>                   initial, 0-3f:4, e:1.s, f:0.s, 40:3, 41-fe:2, ff:4, cc:5, ce:5, f5:5, f7:5
+<icu:state>                   0-40:1.i, 41-fe:1., ff:1.i
+<icu:state>                   0-ff:1.i, 40:1.
+<icu:state>                   0-ff:1.i
+<icu:state>                   0-40:1.i, 41-fe:1.p, ff:1.i
```

Comment:
Why did the old table not need an explicit state table, but the new one does?
Did you try it without this explicit table?
I have a dim memory that genmbcs knows what to do with "EBCDIC_STATEFUL".

Reply:
My understanding is that it did not have surrogate pair mappings. And "EBCDIC_STATEFUL" is sufficient. I got the proposed change from Japan without state table and tried to build it. It ended up an error. After the investigation, it requires explicit state table when some codes mapped to supplemental planes.